src/api/app/assets/javascripts/webui2/cm2/use-codemirror.js

```diff
@@ -9,6 +9,12 @@ window.onbeforeunload = function() {
   }
 }
 
+function cm_mark_diff_lines(cm) {
```

Comment:
The parameter seems unused

Reply:
It's passed in the event case and in line 70 and 79 are other examples of callbacks bearing unused parameters. But as I call the function also without event, I guess javascript doesn't care :)

Reply:
I would drop them there as well. But that's a different story:-)

> I guess javascript doesn't care

Yes, js is flexible in that regard. But why defining variables or parameters we never use:wink: